r/R/rzeppelin/rscala.Rproj

```diff
@@ -0,0 +1,19 @@
+Version: 1.0
```

Comment:
I thought we addressed this way back?  Yes, this does not need to be committed to source, the file will not be there on the next push.